src/reference/torii-endpoints.md

```diff
@@ -172,37 +313,77 @@ view_changes 0
 
 :::
 
-Learn [how to use metrics](/guide/advanced/metrics).
+::: info
+
+To learn more about metrics, see [Metrics](../guide/advanced/metrics.md).
+
+:::
+
+## Pending Transactions
+
+- **Protocol**: `HTTP`
+- **Method**: `GET`
+- **Encoding**: `SCALE`
+- **Endpoint**: `/pending_transactions`
+
+#### Requests
+
+A `GET` request to the endpoint.
+
+#### Responses
+
+| Code | Response | Description                                                                                                                                                          |
+| :--: | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 200  | OK       | Returns a list of pending transactions as [`SignedTransaction`](data-model-schema.md#signedtransaction) objects encoded with `SCALE`; must be decoded by the client. |
+
+**Example**:
+
+```[SCALE]
+[
+  34,
+  86,
+  97,
+  108,
+  117,
+  101,
+  34
+]
+```
```

Comment:
Providing an example here is not useful. I would omit it entirely.

**Note:** providing SCALE example for the Status endpoint still makes sense.